beacon-chain/core/electra/churn.go

```diff
@@ -0,0 +1,85 @@
+package electra
+
+import (
+	"context"
+
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/v5/math"
+	"github.com/prysmaticlabs/prysm/v5/time/slots"
+)
+
+// ComputeConsolidationEpochAndUpdateChurn
```

Comment:
Not sure if it's intended but the Godoc comment looks incomplete

Reply:
Added more